mesh/mesh.cpp

```diff
@@ -4104,6 +4249,18 @@ Mesh::Mesh(const Mesh &mesh, bool copy_nodes)
    mesh.attributes.Copy(attributes);
    mesh.bdr_attributes.Copy(bdr_attributes);
 
+   {
+      // Copy attribute and bdr_attribute names
+      for (auto const &attr_set : mesh.attr_sets)
+      {
+	 attr_set.second.Copy(attr_sets[attr_set.first]);
+      }
+      for (auto const &bdr_attr_set : mesh.bdr_attr_sets)
+      {
+	 bdr_attr_set.second.Copy(bdr_attr_sets[bdr_attr_set.first]);
+      }
+   }
```

Comment:
I'm not sure and I did not try that. I noticed that all of the other Array objects are copied using Array::Copy so I wanted to use the same mechanism here.

Reply:
I think it should work, and since this pattern appears in a number of places, it should simplify things quite a bit.

Note that the copy assignment operator for `Array<T>` is defined using `Array<T>::Copy`, so it should be essentially identical
```cpp
Array<T> &operator=(const Array<T> &src) { src.Copy(*this); return *this; }
```

Reply:
The use of `Copy` here probably dates back to when `Array` did not have copy-assignment.